src/helpers/git_helpers/remote_helpers.sh

```diff
@@ -18,3 +18,12 @@ function remote_repository_name {
   local domain=$(remote_domain)
   remote_url | sed -E "s#.*$domain[/:](.+)#\1#" | sed "s/\.git$//"
 }
+
+# Returns true if a remote repository exists
+function has_remote_url {
+  if [ ! "$(remote_url)" ]; then
```

Comment:
I'd prefer the following (more consistent with the rest of the checks throughout the codebase)
```bash
if [ -z "$(remote_url)"]; then
```
`-z` checks if a string is non-empty